dbt/macros/tests/test_format_additional_select_columns.sql

```diff
@@ -0,0 +1,83 @@
+{% macro test_format_additional_select_columns() %}
+    {% do test_format_additional_select_columns_string_element() %}
+    {% do test_format_additional_select_columns_string_list() %}
+    {% do test_format_additional_select_columns_dict_element_no_column() %}
+    {% do test_format_additional_select_columns_dict_element_no_alias() %}
+    {% do test_format_additional_select_columns_dict_element_with_alias() %}
+    {% do test_format_additional_select_columns_dict_element_array_agg() %}
+    {% do test_format_additional_select_columns_dict_element_array_w_alias() %}
+{% endmacro %}
+
+{% macro test_format_additional_select_columns_string_element() %}
+    {{
+        assert_equals(
+            "test_format_additional_select_columns_string_element",
+            format_additional_select_columns(["foo"]),
+            "foo",
+        )
+    }}
+{% endmacro %}
+
```

Comment:
**suggestion:** Could probably add a test case for the `foo as bar` case:

```suggestion

{% macro test_format_additional_select_columns_string_element() %}
    {{
        assert_equals(
            "test_format_additional_select_columns_string_element",
            format_additional_select_columns(["foo as bar"]),
            "foo as bar",
        )
    }}
{% endmacro %}

```

Reply:
That makes sense, added in 3109ebac03f62c9bfaca78c28c2151b41d474b65.